pulsar-client-cpp/docker-build.sh

```diff
@@ -41,4 +41,4 @@ DOCKER_CMD="docker run -i -v $ROOT_DIR:/pulsar $IMAGE"
 find . -name CMakeCache.txt | xargs rm -f
 find . -name CMakeFiles | xargs rm -rf
 
-$DOCKER_CMD bash -c "cd /pulsar/pulsar-client-cpp && cmake . $CMAKE_ARGS && make check-format && make"
+$DOCKER_CMD bash -c "cd /pulsar/pulsar-client-cpp && cmake . $CMAKE_ARGS && make format && make check-format && make"
```

Comment:
`make format` should not be performed always (though it would be nice to expose it as a script that runs inside docker to do it)

Reply:
that was merged by mistake. revert it.